portfolio/src/main/webapp/index.html

```diff
@@ -32,15 +32,21 @@ <h2>STEP Intern</h2>
         </div>
         </div>
     </div>
-    <section id="about-me">
+    <div="container-fluid" id="about">
+            <h1 class="about-title">About Me</h1>
+            <div class="about-text"><center>
```

Comment:
I think there may be a way to achieve <center> in the css rather than with the center tag if your goal is to center the div.

Reply:
Thank you for your suggestion. I will research more on how to achieve that.